docs/Reference/powerstates.md

```diff
@@ -35,7 +35,14 @@ Commands 100 and above use a combination of HW level power controls (i.e., 2, 5,
 
 ## In-band Required
 
-The power actions below require an in-band agent or Intel® Integrated Management and Security Status (Intel® IMSS).
+The power actions below require an in-band agent, Local Management Service (LMS), or Intel® Integrated Management and Security Status (Intel® IMSS).
+
+LMS is a service that runs locally on an Intel AMT device and enables local management applications to send requests and receive responses to and from the device. The LMS  listens for and intercepts requests directed to the Intel AMT local host and routes them to to the Management Engine via the Intel Management Engine Interface (MEI) driver.
+
+!!! Info "Installing LMS and Drivers"
+    The service installer is packaged with the Intel MEI drivers on the OEM websites. If Windows OS is reimaged or reinstalled, only the Intel MEI Driver is reinstalled, not LMS or IMSS.
+    
+    If the LMS is not installed, go to the OEM website and download the Intel CSME driver. It is usually under the Chipset Driver category.
```

Comment:
I think the CSME driver and IMSS are different packages on most OEM web sites.  We should refer to this as the IMSS or Intel Management and Security Status